projects/plugins/jetpack-boost/bin/delete-release-branch.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Shouldn't be needed in the monorepo.

Reply:
Deleted https://github.com/Automattic/jetpack/pull/20512/commits/20530e155c219bc6baf3899c5469f03dabbe4025